Phoenix/hello/lib/hello/catalog/category.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Hello.Catalog.Category do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "categories" do
+    field :title, :string
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(category, attrs) do
+    category
+    |> cast(attrs, [:title])
```

Comment:
@samu975 could you explain what this line does?

Reply:
Cast takes three parameters. First a struct, in this case I think is "category". Second some attributes, in this case, attrs. And the last one is a list of attributes that are allowed to be altered, in this case is the title. These lines prevent some fields from being updated.